src/Core/Geographic/Extent.js

```diff
@@ -90,7 +92,34 @@ Extent.prototype.as = function as(crs) {
     assertCrsIsValid(crs);
 
     if (_isTiledCRS(this._crs)) {
-        throw new Error('Unsupported yet');
+        if (this._crs == 'WMTS:PM') {
```

Comment:
Can you add something describing the intent here - which seems to be: convert `this` to the requested crs by using `4326` as an intermediate state.